src/components/project/settings/APIKey.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Card, CardText, CardTitle, TextField, RaisedButton } from 'material-ui'
+import Theme from '../../../../style/theme'
+
+const styles = {
+  container: {
+    width: '100%',
+    margin: Theme.padding.tiny
+  }
+}
+
+class APIKey extends React.PureComponent {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      isRevealed: false
+    }
+
+    this.onClickRevealButton = this.onClickRevealButton.bind(this)
+  }
+
+  componentDidMount() {
+
+  }
+
+  onClickRevealButton() {
+    const { isRevealed } = this.state
+    this.setState({ isRevealed: !isRevealed })
+  }
+
+  renderContent() {
+    const { isRevealed } = this.state
+    return (
+      <div>
+        <TextField
+          disabled
+          id="text-field-disabled"
+          type={isRevealed ? 'text' : 'password'}
+          value={this.props.apiKey}
+        /><br />
```

Comment:
Don't use break statements. I know it's being used in some places, it's not good practice. Breaks are meant for separating sentences in text blocks. Use css styles. Padding or margin is fine. Styling is flexible, breaks aren't.

Reply:
I think that we should do a mass fix for this at some other time than. Keep it consistent for now.

Reply:
Not really consistent since it's used only in some places. I've been fixing them as I go. Let's not add more. If you don't want to work with styles, that's fine, removing the break won't affect the ui too much, I can add spacing later. Otherwise take a look at profile settings. All you need is a margin top on the button.

Reply:
fixed. it actually required adding something to the div, not the button.